### PR TITLE
SMS lambda and dockerfile:  Alex

### LINKED ISSUE #62: Dockerise Hourly SMS checker
As a developer, I want the entire stargazing pipeline to be containerized using Docker, so I can ensure consistent environments across all deployments, making the app easier to run and deploy.

- Create `Dockerfile`
- Build Docker image
- Push to ECR

### PR DESCRIPTION
**Pull request by AlexOBarnes**
This closes #62 #64 #50 

What I have done:
- Updated requirements.txt
- Created lambda handler for the sms checker
- Created the associated dockerfile

### CODE REVIEW

File: sms-checker/notification_lambda.py [hunk 1]
```diff
@@ -7,7 +7,13 @@ def lambda_handler(event,context) -> None:
     '''Runs the notification pipeline'''
     logger = logging.getLogger(__name__)
     logging.basicConfig(level=logging.INFO)
-
+    subscribers = get_subscribers()
+    email_list = [entry for entry in subscribers if entry['email']]
+    phone_list = [entry for entry in subscribers if entry['phone']]
+    if email_list:
```
Comment (MurasatoNaoya) [inline sms-checker/notification_lambda.py:13]: Wow, actually sends emails / or text messages. This truly is a work of genius, nay, work of the Gods themselves.

File: sms-checker/notification_lambda.py [hunk 2]
```diff
@@ -7,7 +7,13 @@ def lambda_handler(event,context) -> None:
...
+        send_email(email_list)
+    if phone_list:
+        send_sms(phone_list)
 
 if __name__ == '__main__':
```
Comment (MurasatoNaoya) [inline sms-checker/notification_lambda.py:18]: Works for testing purposes, but Dan made a point last week of removing all things that are unnecessary or 'extra' to our final functionality. I wonder if this falls into that category.

File: sms-checker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+WORKDIR ${LAMBDA_TASK_ROOT}
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt 
+
+COPY notification_lambda.py .
+COPY checker.py .
+COPY message.py .
+
+CMD [ "notification_lambda.lambda_handler" ]
```
Comment (MurasatoNaoya) [inline sms-checker/Dockerfile:12]: Functional docker container for ECR that  imports all necessary files and installs dependencies while running the actual pipeline in the end. LGTM.

**Review by MurasatoNaoya — APPROVED** [review]
LGTM, one pipeline done.

**Review by CalebMathers — APPROVED** [review]

File: sms-checker/requirements.txt [hunk 4]
```diff
@@ -2,5 +2,5 @@ pytest
 pylint
 boto3
 python-dotenv
-psycopg2
+psycopg2-binary
```
[the remark this answers is not in the document]
Comment (MurasatoNaoya) [reply sms-checker/requirements.txt:5]: Agreed. I have nightmares of trying to install base psycopg2 in a container.

**Pull request merged**